__tests__/integration.test.js

```diff
@@ -153,6 +153,78 @@ describe('POST/api/articles/:article_id/comments', () => {
     });
 });
 
+describe('PATCH: api/articles/:article_id', () => {
+    test('200: identifies article id and adds to an articles votes value when given integer', () => {
+        const articleVotes = {
+            inc_votes: 8
+        };
+        return request(app)
+        .patch('/api/articles/3')
+        .send(articleVotes)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.votes).toBe(8)
+            expect(body).toHaveProperty("author");
+            expect(body).toHaveProperty("title");
+            expect(body).toHaveProperty("article_id");
+            expect(body).toHaveProperty("topic");
+            expect(body).toHaveProperty("created_at");
+            expect(body).toHaveProperty("votes");
+            expect(body).toHaveProperty("article_img_url");
+        });
+    });
+    test('200: identifies article id and subtracts from an articles votes value when given a negative number', () => {
+        const articleVotes = {
+            inc_votes: -15
+        };
+        return request(app)
+        .patch('/api/articles/3')
+        .send(articleVotes)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.votes).toBe(-15)
+            expect(body).toHaveProperty("author");
+            expect(body).toHaveProperty("title");
+            expect(body).toHaveProperty("article_id");
+            expect(body).toHaveProperty("topic");
+            expect(body).toHaveProperty("created_at");
+            expect(body).toHaveProperty("votes");
+            expect(body).toHaveProperty("article_img_url");
+        });
+    });
```

Comment:
Really good test as well, but same as the above this should also be a 200 status code.